src/forms/FormFieldCheckbox/FormFieldCheckbox.stories.tsx

```diff
@@ -1,118 +1,116 @@
-// import * as React from "react";
-// import { useMemo, ReactElement } from "react";
-// import { boolean, withKnobs, text } from "@storybook/addon-knobs";
-// import { Meta } from "@storybook/addon-docs/blocks";
-// import { FormFieldCheckboxDef } from ".";
-// import { FieldDef } from "@root/components/Field";
+import * as React from "react";
+import { useMemo } from "react";
+import { FormFieldCheckboxDef } from ".";
+import { FieldDef } from "@root/components/Field";
 
-// // Components
-// import Form, { useForm } from "@root/components/Form";
+// Components
+import Form, { useForm } from "@root/components/Form";
 
-// // Utils
-// import { checkboxOptions } from "./FormFieldCheckboxUtils";
-// // import { onCancel, renderButtons } from "@root/utils/storyUtils";
+// Utils
+import { checkboxOptions } from "./FormFieldCheckboxUtils";
+import { excludedFormFieldsControls, onCancel, renderButtons } from "@root/utils/storyUtils";
 
-// export default {
-// 	title: "FormFields/FormFieldCheckbox",
-// 	decorators: [withKnobs],
-// } as Meta;
+export default {
+	title: "FormFields/FormFieldCheckbox",
+	component: Form,

```

Comment:
Almost in all stories I saw that component was Form, but I think we should be using their respective field (in this case FormFieldCheckbox).... the objective would be to fully show the inputSettings in the knobs, for example, for this component we should have "options", just like how we have the formFieldText